### DISCUSSION TITLE
Collecting embeddings using towhee

### DISCUSSION THREAD

**Discussion by ZubeenK** [towhee-io/towhee#2505] [Q&A] [answered]
from PIL import Image
from towhee.dc2 import pipe, ops
import numpy as np

vector_embeddings = []

p = (pipe.input('target')
        .map('target', 'target_emb', ops.image_embedding.timm(model_name='resnet50'))
        .output('target_emb')
)

for image in images_list:
    np_image = np.array(image)
    res = p(np_image)
    DataCollection(res).show()
    
I am able to visualize 'target_emb' using above code, but I want to collect 'target_emb' in a list. How can I do this?

**Comment by junjiejiangjjj** [comment, marked as answer]
res.get() or res.get_dict()

**Reply by ZubeenK** [reply]
Thanks for replying, it worked for me.
